infra/compose/.env.images.example

```
@@ -52,3 +52,11 @@ COMPOSIO_API_KEY=
 # HTTP_PROXY=
 # HTTPS_PROXY=
 # NO_PROXY=localhost,127.0.0.1,postgres,supervisor,api,worker,web
+
+# Structured logs. JSON in production. Axiom is optional and disabled unless both are set.
+LOG_LEVEL=info
+LOG_FORMAT=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The LOG_FORMAT key should go before the LOG_LEVEL key

(UnorderedKey)